contrib/platform/test/com/sun/jna/platform/unix/X11Test.java

```diff
@@ -116,6 +116,25 @@ public void testXGetWMProtocols() {
         Assert.assertArrayEquals("Sent protocols were not equal to returned procols for XGetWMProtocols", sentAtoms, receivedAtoms);
     }
 
+    public void testXQueryExtension() {
+        final IntByReference opcode = new IntByReference(0);
+        final IntByReference first_event = new IntByReference(0);
+        final IntByReference first_error = new IntByReference(0);
+
+        // check if the XTEST extension is available
+        if (X11.INSTANCE.XQueryExtension(display, "XTEST", opcode, first_event, first_error)) {
+            // Opcode for extension should be assigned in range 128-255
+            Assert.assertTrue("Wrong value for opcode returned", opcode.getValue() >= 128);
```

Comment:
(Style preference) The message string should say what should happen, e.g., "Value for opcode should be between 128-255."  

Also, you could combine both this and the next line with `assertTrue(opcode.getValue() & 0x80 > 0)`.

Reply:
done